Improving the Login UI

luquiceno opened the pull request:
Using Flowbite an open source library, I improved the login UI. And I changed the structure of the code a little, adding the components folder.

luquiceno committed:
Improving the Login UI
Using Flowbite an open source library, I improved the login UI.
And I changed the structure of the code a little, adding the components
folder.

sebasutp commented on src/App.css:
Don't comment out unnecessary code, just remove it. Less code makes it easier for people to navigate and understand it. We can always bring the code back if we need to using git :)

sebasutp commented on src/App.jsx:
```diff
@@ -1,11 +1,12 @@
-import { useState } from 'react'
-import { BrowserRouter, Routes, Route, Navigate, useNavigate } from 'react-router-dom';
+//import { useState } from 'react'
```
Delete

luquiceno replied:
Thanks for your feedback.

sebasutp commented on src/index.css:
```diff
@@ -1,4 +1,8 @@
+@tailwind components;
+@tailwind utilities;
+
+/* :root {
```
Delete

sebasutp commented on src/main.jsx:
```diff
@@ -1,3 +1,4 @@
+//import { StrictMode } from 'react'
```
Delete

sebasutp reviewed (commented):
Looks good. I left some comments with the issues. In addition to that, please add a screenshot to the pull request showing how the changes look like in the frontend

luquiceno commented:
This is a screenshot to see how looks until now the UI Login.

<img width="778" alt="Screenshot_1" src="https://github.com/user-attachments/assets/98dbefae-0a45-4375-a311-2ac49c1e35a0">

luquiceno committed:
Resolving suggestions
I deleted the commented code and I added a screenshot to see how looks
the UI until now.
